.travis.yml

```diff
@@ -1,21 +1,38 @@
 sudo: required
 dist: trusty
 language: node_js
-
 node_js:
-  - 6
+  - '6'
+
+addons:
+apt:
+  sources:
+    - google-chrome
+  packages:
+    - google-chrome-stable
+    - google-chrome-beta
+
+before_install:
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+before_script:
+  - npm install -g karma
+  - npm install
+  - npm run build
 
 script:
   - npm test
+  - karma start config/karma.conf.js --single-run
```

Comment:
here you are invoking karma script with config file in config/karma.conf.js, the file is not present there in the current state of repository. Please add those files as well in the PR or no do test with karma for now.